src/reducers/index.js

```diff
@@ -0,0 +1,40 @@
+import { combineReducers } from "redux";
+import * as actions from "../actions";
+import model from "../resources/model.js";
+
+const currentStateReducer = (state = { status: "idle", value: "" }, action) => {
```

Comment:
Okay so your naming here is really hard to follow.

We have a reducer called currentStateReducer, exported under the name currentState.

The state tree looks like this:
```
state: {
  currentState: {
    status,
    value
  },
  ctas: ??
  messages: ??
}
```

When reviewing react I start with the reducers as normally the business logic lives, but its very hard to see what any of this means..